src/main/webapp/js/finscholar/commonlistview/commonlistview.soy

```diff
@@ -19,10 +19,10 @@
   <div id="scroll-div">
     <ul id="table-body" class="mdl-list">
     </ul>
+  </div>
```

Comment:
Can you make endofList included in scroll-div? I added some styles to it in my branch.